zerver/lib/subscription_info.py

```diff
@@ -36,12 +37,23 @@
     SubscriptionInfo,
     SubscriptionStreamDict,
     UserGroupMembersDataDict,
+    UserGroupMembersTypedDict,
 )
 from zerver.lib.user_groups import UserGroupMembershipDetails, get_recursive_membership_groups
 from zerver.models import Realm, Stream, Subscription, UserProfile
 from zerver.models.streams import get_all_streams
 
 
+def bulk_convert_to_user_group_members_typed_dict(
```

Comment:
Not sure if bulk is the right word to use here or do we only reserve it for functions dealing with the database